cur/programming/lists/hof/ttt/05-winning-square.html

```diff
@@ -22,14 +22,19 @@
 </blockquote>
 <hr />
 <p>Okay, time to move from planning to doing. The next goal is to build a <code>winning square</code> block. It'll take a list of triples and an X-or-O, and look for a square in which that player can move to win immediately. What does that mean in terms the program can understand? (We asked you to think about this a few screens ago.) Our answer is, &quot;look for a winning combination in which our letter appears twice and the other letter doesn't appear at all.&quot; So we seem to need two helpers: one that reports how many times a letter appears in a triple, and one that reports &quot;the other letter.&quot;</p>
-<p>Start with the second of those. Write a reporter called <code>opponent</code> that just takes an <code>X</code> or <code>O</code> as its input, and reports the other letter.<br />
+<div class="alert quoteGreen alert-success">
+	<p>Start by writing a reporter called <code>opponent</code> that just takes an <code>X</code> or <code>O</code> as its input, and reports the other letter.<br />
```

Comment:
I'm going to replace the starter file later tonight, should I remove the block or keep this section?

This block is honestly pretty trivial, so it doesn't seem worth having people write it.